components/empty-state/empty-state-action-button.js

```diff
@@ -5,6 +5,7 @@ import { css, html, LitElement, nothing } from 'lit';
 /**
  * `d2l-empty-state-action-button` is an empty state action component that can be placed inside of the default slot of `empty-state-simple` or `empty-state-illustrated` to add a button action to the component.
  * @fires d2l-empty-state-action - Dispatched when the action button is clicked
+ * @fires d2l-empty-state-illustrated-check - Internal event
```

Comment:
Do you want this event to show up in the docs?  If not, then it can be ignored by adding `/** @ignore */` before the dispatch statement. For [example](https://github.com/BrightspaceUI/core/blob/main/components/list/list.js#L251).

Reply:
This won't work. The `web-component-analyzer` will only parse `/** @ignore */` if you immediately dispatch the event. In my case I saved it as a variable  and the `/** @ignore */` won't work. That's why the `Internal event` case was hardcoded into Daylight in situations where the analyzer does not work as expected. You can see this used in selection-input [here](https://github.com/BrightspaceUI/core/blob/04668717fca0b18c789c8f46c67471b7f2c1a1b9/components/selection/selection-input.js#L16).

That's what prompted me to update the docs in [this PR](https://github.com/BrightspaceUI/documentation/pull/986).